crates/core/src/operations/merge/filter.rs

```diff
@@ -252,16 +252,13 @@ pub(crate) fn generalize_filter(
             }
         }
         Expr::InList(in_list) => {
-            let compare_expr = match generalize_filter(
+            let compare_expr = generalize_filter(
```

Comment:
Why is this changed?

Reply:
I fixed a few warnings along the way, this statement could be just written with an `?` operator instead.